### PR TITLE
Fix/batching

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

**Comment by caoimheb** [comment]
Ready for review, but don't merge until `worker` and `web` are also good to go

File: app/points/domain/workload.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Workload {
     this.licenseTiers = licenseTiers
     this.licenseCasesLast16Weeks = licenseCasesLast16Weeks
     this.communityCasesLast16Weeks = communityCasesLast16Weeks
+    this.stagingId = stagingId
```
Comment (ddebella) [inline app/points/domain/workload.js]: Are you validating the stagingId property?

File: app/staging/domain/om-workload.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class OmWorkload {
-  constructor (casesSummary, courtReports, instReports, caseDetails, workloadId) {
+  constructor (stagingId, casesSummary, courtReports, instReports, caseDetails, workloadId) {
+    this.stagingId = stagingId
```
Comment (andrew-js-wright) [inline app/staging/domain/om-workload.js]: I guess this is just `id` in staging object.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wmt-probation-rules",
-  "version": "0.1.12",
+  "version": "0.1.13",
```
Comment (andrew-js-wright) [inline package.json]: We're going to need to coordinate our versioning here.

Comment (caoimheb) [reply package.json]: Yeh, I just bumped from `master`, but I'm guessing your PR will be in first, so I'll merge and rebump

File: app/points/domain/workload.js [hunk 4]
```diff
@@ -20,6 +20,8 @@ class Workload {
...
+    this.workloadReportId = workloadReportId
```
Comment (andrew-js-wright) [inline app/points/domain/workload.js]: There should be a unit test that makes sure when this is set through the constructor it is assigned to the correct internal variable.

File: test/unit/points/domain/test-workload.js [hunk 5]
```diff
@@ -38,8 +42,15 @@ describe('points/domain/Workload', function () {
 
   it('all fields can be retrieved', function () {
     var workload = pointsHelper.getTestWorkloadObject()
+    expect(workload.workloadOwnerId).to.eql(workloadOwnerId)
+    expect(workload.monthlySdrs).to.eql(monthlySdrs)
+    expect(workload.sdrsDueNext30Days).to.eql(sdrsDueNext30Days)
+    expect(workload.paromsCompletedLast30Days).to.eql(paromsCompletedLast30Days)
+    expect(workload.paromsDueNext30Days).to.eql(paromsDueNext30Days)
```
Comment (andrew-js-wright) [inline test/unit/points/domain/test-workload.js]: Thanks for this!

**Pull request merged**